docs/guides/setup-channel.mdx

```diff
@@ -121,20 +112,61 @@ print(result)
   ```
 
 <span id="iqp-rest-api"></span>
-## Set up to use IBM Quantum Platform with REST API
+### Set up to use IBM Quantum Platform with REST API
 
 Alternatively, you can also access quantum processors with REST APIs, enabling you to work with QPUs using any programming language or framework. 
 
-1. Retrieve your IBM Quantum token from the [IBM Quantum account page](https://quantum.ibm.com/account).
-
-2. If you do not already have a user account, get one at the [IBM Quantum login page.](https://quantum.ibm.com/login)  Your user account is associated with one or more [instances](./instances) (in the form hub / group / project) that give access to IBM Quantum services. Additionally, a unique token is assigned to each account, allowing for IBM Quantum access from Qiskit. The instructions in this section use our default instance.  For instructions to choose a specific instance, see [Connect to an instance](./instances#connect-instance).
+1. If you do not already have a user account, get one at the [IBM Quantum login page.](https://quantum.ibm.com/login)  Your user account is associated with one or more [instances](./instances) (in the form hub / group / project) that give access to IBM Quantum services. Additionally, a unique token is assigned to each account, allowing for IBM Quantum access from Qiskit. The instructions in this section use our default instance.  For instructions to choose a specific instance, see [Connect to an instance](./instances#connect-instance).
 
     <Admonition type="note">
       The Instances section in your [IBM Quantum account page](https://quantum.ibm.com/account) lists the instances that you can access.
     </Admonition>
 
+1. Retrieve your API token from the [IBM Quantum account page](https://quantum.ibm.com/account).
+
+1. Because of the security risks posed by executing code that contains your API token in plain text, the recommended authentication method is to first create an environment variable for your API token, but only do so **if you are working in a trusted Python environment (such as on a personal laptop or workstation).** [Skip to the next step](#rest-untrusted) if you are not using a trusted environment, such as a shared or public computer.
+
+    The following demonstrates creating an environment variable:
```

Comment:
for a trusted device, we can use the original wording - _Set the IQP_API_TOKEN environment variable in your system. You can do this by adding the following line to your shell profile (e.g., .bashrc, .zshrc) or by setting it directly in your terminal:_

Reply:
resolved by 1f3d40de4d4fdcde7f47199b13ace582c4466670